src/SourceLocator/Ast/Parser/MemoizingParser.php

```diff
@@ -30,7 +30,7 @@ public function __construct(Parser $wrappedParser)
     /**
      * {@inheritDoc}
      */
-    public function parse(string $code, ?ErrorHandler $errorHandler = null) : ?array
+    public function parse($code, ?ErrorHandler $errorHandler = null)
```

Comment:
What is the signature of the parent here? Why is it complaining about incompatibility exactly? Was there an upstream change that broke this?